### PR TITLE
add merchant_payout model

### PR DESCRIPTION
**Pull request by juliexxia**
No tests yet - please review for model content! 

TODO
- [ ] tests
- [ ] doc fields (in schema.rb?)

### CODE REVIEW

File: db/migrate/20200502130958_create_merchant_payouts.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class CreateMerchantPayouts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :merchant_payouts do |t|
+      t.integer :payout_type, default: 0, null:false
+      t.integer :total_amount, null: false
+      t.integer :account_number, null: false
+      t.date :payment_delivered
```
Comment (jtmckibb) [inline db/migrate/20200502130958_create_merchant_payouts.rb:7]: Could we add a "note" field?

File: db/migrate/20200502130958_create_merchant_payouts.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class CreateMerchantPayouts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :merchant_payouts do |t|
+      t.integer :payout_type, default: 0, null:false
+      t.integer :total_amount, null: false
+      t.integer :account_number, null: false
```
Comment (jtmckibb) [inline db/migrate/20200502130958_create_merchant_payouts.rb:6]: Not sure we wanna keep our account numbers in our DB. Could this be a string instead? Right now we have two bank accounts we're paying out from.

**Comment by Yong-L** [comment]
this might make it easier for you to annotate the models: https://github.com/ctran/annotate_models

**Comment by jacoblb64** [comment]
Closing this as it's been way out of date

**Pull request closed without merging**